README.md

```diff
@@ -4,8 +4,19 @@
 
 [![build-status](https://github.com/SaranjeetKaur/isc_proposal_translations/actions/workflows/publish-proposal.yaml/badge.svg)](https://github.com/SaranjeetKaur/isc_proposal_translations/actions/workflows/publish-proposal.yaml)
 
-This repository contains the proposal for "Consistent Translations Glossary across R". The proposal aims to improve the technical infrastructure of the translations process for R packages. The goal is to provide a common shared glossary across the different R translations projects. It will help in improving the quality of translations and make it easier for users to contribute to the translations process. This will help in making R more accessible to users who do not speak English and improve the overall user experience of R. Feedback on the proposal is welcome - please feel free to provide your feedback by opening an issue in this repository.
+This repository contains the proposal for "Consistent Translations Glossary across R". The proposal aims to improve the technical infrastructure of the translations process across R projects. There are many volunteer translation efforts across the R community (messages in base R/recommended packages, rOpenSci materials such as their package development guide, Bioconductor training materials, R books such as R for Data Science, etc). These projects often have their own glossaries, since the benefits of using a glossary in translation projects are widely recognised (improves translation quality, reduces translation effort). However, developing and maintaining separate glossaries for each translation project reduces these benefits - different projects may use different translations for certain terms, effort to identify relevant terms and translate them is duplicated across projects putting unnecessary burden on volunteers. Some projects do not have the volunteer capacity to translate the glossary into multiple languages, or may not even use a glossary. Currently it is difficult for projects to take full advantage of existing glossaries due to diversity in the choice of translation infrastructure. These issues will become even more important if internationalization of R help pages becomes more common (ref R Consortium-funded project: [tooling-for-internationalization-of-r-help-pages](https://r-consortium.org/all-projects/2023-group-2.html#tooling-for-internationalization-of-r-help-pages))
 
+An example of diverging translation of glossary terms is provided by the rOpensci English-Spanish glossary:
+
+| English | Spanish (rOpenSci) | Spanish (R project)                        |
+|--------|--------------------|--------------------------------------------|
+| path    | ruta de acceso     | ruta (file path) / trayecto (polygon path) |   
```

Comment:
Actually, the `polygon` part is just translating `path` into Spanish in the wrong context, I would say... But it s a very good example.